policyengine_us/variables/gov/states/az/tax/income/deductions/standard/az_increased_standard_deduction.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class az_increased_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona increased standard deduction"
+    unit = USD
+    documentation = "https://www.azleg.gov/viewdocument/?docName=https://www.azleg.gov/ars/43/01041.htm"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.az.tax.income.deductions.standard
+        charitable_deduction = tax_unit("charitable_deduction", period)
+        charitable_contributions_credit = tax_unit(
```

Comment:
Where does it say to reduce by the credit? It's not on the worksheet of https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140BOOKLET.pdf#page=27

Reply:
We interpreted the passage below since the charitable deduction in AZ includes a reduction of the federal charitable deduction by the credit

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/ccfb247d-1b64-4045-9c55-c76355dbf038)

Reply:
Oh I missed line 5C which is the credit that's subtracted, thanks